requirements.in

```diff
@@ -2,15 +2,15 @@ apscheduler~=3.6
 Click~=7.0
 cachetools~=3.1
 dictdiffer
```

Comment:
missing version?

Reply:
Yeah, I see this. Should we set it `~=0.8`, as it is `dictdiffer==0.8.0` in `requirements.txt`
@epa095